magento/pwa-studio
# [Doc] Publish 4.0.0 draft docs

## Description

Revise and move the 4.0.0 draft docs out of the `_drafts` folder so they can be published.

## Related Issue

Closes #1679 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run linter on the following files (`npm run lint <filepath>`)

    * `src/pwa-buildpack/configuration-management/index.md`
    * `src/pwa-buildpack/reference/configure-webpack/index.md`
    * `src/pwa-buildpack/reference/buildpack-cli/create-custom-origin/index.md`
    * `src/pwa-buildpack/reference/buildpack-cli/create-env-file/index.md`
    * `src/pwa-buildpack/reference/buildpack-cli/index.md`
    * `src/pwa-buildpack/reference/buildpack-cli/load-env/index.md`

    Everything else is auto-generated content, so you don't need to lint those files.

3. Verify no linting errors for these files.
4. Build the HTML preview: `npm run start` 
5. Verify site successfully builds
6. Verify the following pages are navigable and have a table of content entry:

    * `/peregrine/reference/hooks/useEventListener/`
    * `/peregrine/reference/hooks/usePagination/`
    * `/peregrine/reference/toasts/useToasts/`
    * `/peregrine/reference/toasts/useToastContext/`
    * `/peregrine/reference/hooks/useWindowSize/`
    * `/pwa-buildpack/configuration-management/`
    * `/pwa-buildpack/reference/buildpack-cli/`
    * `/pwa-buildpack/reference/buildpack-cli/create-custom-origin/`
    * `/pwa-buildpack/reference/buildpack-cli/create-env-file/`
    * `/pwa-buildpack/reference/buildpack-cli/load-env/`
    * `/pwa-buildpack/reference/configure-webpack/`
    * `/pwa-buildpack/reference/environment-variables/`
    * `/venia-ui/reference/components/ToastContainer/`

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
